.circleci/config.yml

```diff
@@ -26,8 +26,8 @@ jobs:
           key: scala-library-dependencies-{{ checksum "build.sbt" }}
       - run:
           command: |
-            sbt clean ++"2.12.10" test
-            sbt clean ++"2.13.1" test
+            sbt clean ++"2.12.10" scalafmtCheckAll scalafmtSbt test
+            sbt clean ++"2.13.1" scalafmtCheckAll scalafmtSbt test
```

Comment:
Do you want to set `scalafmtSbtCheck`?

Reply:
Oops. Fixed in https://github.com/chatwork/scala-jwk/pull/43/commits/de1c85c84ebb1b87f3e7483b04b6686b568748a5